### ISSUE TITLE
Down arrow becomes invisible if button has property Flat set to True and Style set to bsSplitButton

### ISSUE THREAD

**Issue by vectoris** [EtheaDev/StyledComponents#7]
This needs to be fixed. If this is not a bug, then it should be possible to make this arrow visible.

**Comment by carloBarazzetta** [comment]
Try with those changes and give me a feed-back:

``` Delphi
procedure TStyledButtonRender.DrawBackgroundAndBorder(
  const ACanvas: TCanvas;
  const AStyleAttribute: TStyledButtonAttributes);
var
  DrawRect, SplitButtonRect: TRect;
begin
  //Erase Background
  EraseBackground(ACanvas);

  //Don't draw button face for Flat Buttons
  if not (FFlat and (FState in [bsUp, bsDisabled]) and not (FMouseInControl)) or
    Focused then
  begin
    DrawRect := FOwnerControl.ClientRect;

    //Draw Button Shape
    CanvasDrawshape(ACanvas, DrawRect, FStyleDrawType, FStyleRadius);
  end;

  //Draw Bar and Triangle
  if FDropDownRect.Width > 0 then
  begin
    SplitButtonRect.Left := DrawRect.Width - FDropDownRect.Width;
    if FFlat then
      ACanvas.Pen.Style := psClear;
    CanvasDrawBarAndTriangle(ACanvas, FDropDownRect,
      {$IFDEF D10_3+}FOwnerControl.ScaleFactor{$ELSE}1{$ENDIF},
      ACanvas.Pen.Color, AStyleAttribute.FontColor);
  end;
end;
```

**Comment by vectoris** [comment]
These changes solve the problem for the run-time. 
Design-time still not working.

**Comment by carloBarazzetta** [comment]
Fixed in 3.2.0 version
